applications/GeoMechanicsApplication/custom_constitutive/tension_cutoff.hpp

```diff
@@ -0,0 +1,40 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohame Nabi,
```

Comment:
There is a typo in your name:
```suggestion
//  Main authors:    Mohamed Nabi,
```